src/content/docs/en/recipes/bun.mdx

```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
 bun ./dist/server/entry.mjs
 ```
 
+## Testing in Bun
+
+Bun ships with a fast, built-in, Jest-compatible test runner through the `bun test` command. If you like to use that read [`bun test` documentation](https://bun.sh/docs/cli/test).
+
+However, you may want to use Cypress or Playwright for a modern approach to testing web apps.
+
+### Cypress
+
+Cypress is a front-end testing tool and is on a mission to "make the testing experience enjoyable and generate developer happiness". This enables you to write end-to-end tests for your Astro site.
+
+You can install Cypress like so:
```

Comment:
```suggestion
Install Cypress with the following command:
```
Whenever it makes reasonable sense, we prefer to give direct instructions. So, something like the above, or, "To install Cypress, use the following command:"

You have to imagine that the response to "you can..." is always, "I know I *can*... should I? Do I want to? Is there another way?"  Just tell people what to do!  It makes them feel more confident in what's going to happen!